src/inmanta/types.py

```diff
@@ -50,6 +50,14 @@ def validate(cls, value: Any) -> bool:
 
         raise errors.StrictBoolError()
 
+    @classmethod
+    def __modify_schema__(cls, f_schema: Dict[str, Any]) -> Dict[str, str]:
```

Comment:
Does this always return a `Dict[str, str]`? What about `f_schema`'s other fields than `type`? From a typing perspective I find it strange that mypy does not complain about this, since it can't know of other values.